src/ImageSharp.Drawing/Processing/Extensions/ClearPathExtensions.cs

```diff
@@ -1,79 +1,84 @@
 // Copyright (c) Six Labors.
 // Licensed under the Apache License, Version 2.0.
 
-using SixLabors.ImageSharp.Drawing.Processing.Processors.Drawing;
 using SixLabors.ImageSharp.Processing;
 
 namespace SixLabors.ImageSharp.Drawing.Processing
 {
     /// <summary>
-    /// Adds extensions that allow the filling of polygon outlines to the <see cref="Image{TPixel}"/> type.
+    /// Adds extensions that allow the flood filling of polygon outlines without blending.
     /// </summary>
     public static class ClearPathExtensions
     {
         /// <summary>
-        /// Clones the shape graphic options and applies changes required to force clearing.
+        /// Flood fills the image in the path of the provided polygon with the specified brush without any blending.
         /// </summary>
-        /// <param name="shapeOptions">The options to clone</param>
-        /// <returns>A clone of shapeOptions with ColorBlendingMode, AlphaCompositionMode, and BlendPercentage set</returns>
-        internal static DrawingOptions CloneForClearOperation(this DrawingOptions shapeOptions)
-        {
-            GraphicsOptions options = shapeOptions.GraphicsOptions.DeepClone();
-            options.ColorBlendingMode = PixelFormats.PixelColorBlendingMode.Normal;
-            options.AlphaCompositionMode = PixelFormats.PixelAlphaCompositionMode.Src;
-            options.BlendPercentage = 1;
-
-            return new DrawingOptions(options, shapeOptions.ShapeOptions, shapeOptions.TextOptions, shapeOptions.Transform);
-        }
+        /// <param name="source">The image processing context.</param>
+        /// <param name="color">The color.</param>
+        /// <param name="path">The logic path.</param>
+        /// <returns>The <see cref="IImageProcessingContext"/> to allow chaining of operations.</returns>
+        public static IImageProcessingContext Clear(
+            this IImageProcessingContext source,
+            Color color,
+            IPath path) =>
```

Comment:
I would consider renaming these parameters to `region` so the name better represents the role of the parameter. Also, we definitely need more descriptive docs than "The logic path.".

```suggestion
        /// <param name="region">The <see cref="IPath"/> defining the region to fill.</param>
        /// <returns>The <see cref="IImageProcessingContext"/> to allow chaining of operations.</returns>
        public static IImageProcessingContext Clear(
            this IImageProcessingContext source,
            Color color,
            IPath region) =>
```